publishers/sidekiq.go

```diff
@@ -7,20 +7,19 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
-	"os"
 	"time"
 
 	log "github.com/sirupsen/logrus"
 
-	"github.com/go-redis/redis/v8"
 	"github.com/librariesio/depper/data"
+	"github.com/librariesio/depper/redis"
+	_ "github.com/librariesio/depper/redis"
```

Comment:
What's up with this import?

Reply:
Woops, probably gopls weirding things up!